src/Shared/Controls/DebugLogs.xaml

```diff
@@ -0,0 +1,116 @@
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+
+    <StackPanel
+		MaxWidth="1600"
+		Margin="20"
+		Spacing="4" VerticalAlignment="Top">
+
+
+        <TextBlock
+            MaxWidth="1000"
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Debug Info" />
+
+        <labs:SettingsCard
+			Header="Window Size"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Height(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowHeight}"/>
+                    <Run Text=" Width(px):"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.WindowWidth}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:SymbolIcon Symbol=Clear}"
+			Header="Delete User Data"
+			HeaderIcon="{ui:SymbolIcon Symbol=Delete}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.DeleteUserDataCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			Header="Memory"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Memory Used:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.MemoryUsage}"/>
+                    <Run Text="Expected Memory Limit:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.TotalMemory}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
+			Header="Open logs folder"
+			HeaderIcon="{ui:SymbolIcon Symbol=Folder}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.OpenLogFolderCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <TextBlock
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Logs" />
+
+        <ScrollViewer x:Name="PART_SvLogs"
+                      MaxWidth="1600"
+                      MaxHeight="450" HorizontalScrollBarVisibility="Visible" HorizontalScrollMode="Enabled" >
+            <ListView  ItemsSource="{x:Bind AppRoot.AppDebug.AppLogs, Mode=OneWay}" >
+
+                <ListView.ItemsPanel>
+                    <ItemsPanelTemplate>
+                        <VirtualizingStackPanel>
```

Comment:
Let's swap this out for an `ItemsStackPanel`:

```
     <ListView.ItemsPanel>
         <ItemsPanelTemplate>
             <ItemsStackPanel ItemsUpdatingScrollMode="KeepLastItemInView" VerticalAlignment="Bottom">
                 <ItemsStackPanel.ChildrenTransitions>
                    <TransitionCollection />
                  </ItemsStackPanel.ChildrenTransitions>
             </ItemsStackPanel>
         </ItemsPanelTemplate>
     </ListView.ItemsPanel>
```

`ItemsStackPanel` is the default used by ListView and supports virtualization, but the code above also enables a "grow list from bottom" behavior, demonstrated by the WinUI 2 Gallery app:

![image](https://user-images.githubusercontent.com/9384894/209713174-84371b81-1b55-4507-93fd-3bcde7ebe602.png)